bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/RRD4jPersistenceService.java

```diff
@@ -321,8 +324,6 @@ public Iterable<HistoricItem> query(FilterCriteria filter) {
             // do not call method {@link RrdDb#createFetchRequest(ConsolFun, long, long, long)} if start > end to avoid
             // an IAE to be thrown
             if (start > end) {
-                logger.warn("Could not query rrd4j database for item '{}': start ({}) > end ({})", itemName, start,
-                        end);
```

Comment:
Why is this warning removed?

Reply:
Since this can only happen when "internal" calculations make `start > end` don't think it makes sense to log it ... at least not at warn level?

Reply:
I seem to remember some case where it was only milliseconds in difference, but I'm not sure. I'd suggest to keep it as **debug** to be able to backtrack such cases.

Reply:
done